internal/admin/server.go

```diff
@@ -0,0 +1,27 @@
+package admin
+
+import (
+	"net/http"
+	"net/http/pprof"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const DefaultAdminAddr = ":6060"
+
+// DefaultAdminServeMux is the default ServeMux to be used for admin servers in packages like httpbp, thriftbp, etc.
+// DefaultAdminServeMux configures the following routes:
+//    metrics       - serve /metrics for prometheus
+//    profiling     - serve /debug/pprof for profiling, ref: https://pkg.go.dev/net/http/pprof
+var DefaultAdminServeMux = http.NewServeMux()
+
+func init() {
+	// The debug/pprof endpoints follow the pattern from the init function in net/http/pprof package.
```

Comment:
the description is good to have, but I would also (for convenience) provide a link